src/main/python/janusgraph_python/utils/toGeoJSON.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class toGeoJSON(object):
```

Comment:
Why is this serialization implemented in `utils` and not in `serializer` like the other serialization logic? This is a really confusing place. Also the name doesn't hint at the fact that it only applies to GeoShapes.
I would instead create a `PointSerializer` and a `CircleSerializer` and put them into the `serializer` folder.

Reply:
I created #18  to track the same. We can move the code inside toGeoJSON class to each Serializer class. The reason it was not part earlier was because I didn't want to clutter the Serializer classes.